.circleci/config.yml

```
@@ -1,7 +1,7 @@
 version: 2.1
 gpu: &gpu
   machine:
-    image: ubuntu-1604:201903-01
+    image: ubuntu-2004:202107-02
```

```suggestion
    image: ubuntu-2004-cuda-11.2:202103-01
```
This has CUDA 11.2 pre-installed which should simplify the installation if that is accessible. I don't know if a Ubuntu-2004 has a cuda 11.1 preinstalled: https://circleci.com/docs/2.0/configuration-reference/

---

I guess cuda-11.2 may be fine here, but in general cuda-11.2 is avoided since its a buggy slow mess.